pkg/extender/extender.go

```diff
@@ -35,6 +36,8 @@ const (
 	defaultScore = 5
 
 	schedulingFailureEventReason = "FailedScheduling"
+	annotationPrefix             = "stork.libopenstorage.org/"
+	volumerestore                = annotationPrefix
```

Comment:
This should be ` annotationPrefix + "restore-in-progress"`?

Reply:
Actually, this is defined in `pkg/snapshot/controllers/snapshotrestore.go` and should be used from there